src/experiment/trials/camera.js

```diff
@@ -9,111 +9,105 @@ const WEBCAM_ID = "webcam";
+          return;
+        }
+        const cameraChunks = [];
 
-          // Push data whenever available
-          cameraRecorder.addEventListener("dataavailable", (event) => {
-            if (event.data.size > 0) cameraChunks.push(event.data);
-          });
+        // Push data whenever available
+        cameraRecorder.addEventListener("dataavailable", (event) => {
+          if (event.data.size > 0) cameraChunks.push(event.data);
+        });
 
-          // Saves the raw data feed from the participants camera (executed on cameraRecorder.stop()).
-          cameraRecorder.addEventListener("stop", () => {
-            const blob = new Blob(cameraChunks, { type: cameraRecorder.mimeType });
+        // Saves the raw data feed from the participants camera (executed on cameraRecorder.stop()).
+        cameraRecorder.addEventListener("stop", () => {
+          const blob = new Blob(cameraChunks, { type: cameraRecorder.mimeType });
 
-            // Pass video data to Electron as a base64 encoded string
-            const reader = new FileReader();
-            reader.readAsDataURL(blob);
-            reader.onloadend = () => {
-              window.electronAPI.saveVideo(reader.result);
-            };
-          });
-        },
-        on_load: function () {
-          // Assign camera feed to the <video> element
-          const camera = document.getElementById(WEBCAM_ID);
+          // Pass video data to Electron as a base64 encoded string
+          const reader = new FileReader();
+          reader.readAsDataURL(blob);
+          reader.onloadend = () => {
+            window.electronAPI.saveVideo(reader.result);
+          };
+        });
+      },
+      on_load: function () {
+        // Assign camera feed to the <video> element
+        const camera = document.getElementById(WEBCAM_ID);
 
-          camera.srcObject = window.jsPsych.pluginAPI.getCameraRecorder().stream;
-        },
-        on_finish: function () {
-          // Begin video recording
-          window.jsPsych.pluginAPI.getCameraRecorder().start();
-        },
+        camera.srcObject = window.jsPsych.pluginAPI.getCameraRecorder().stream;
+      },
+      on_finish: function () {
+        // Begin video recording
+        window.jsPsych.pluginAPI.getCameraRecorder().start();
       },
-    ],
-  };
-}
+    },
+  ],
+};
 
 /**
  * A trial that finishes recording the participant using their computer's default camera
  *
- * @param {Number} duration How long to show the trial for
  * @returns {Object} A jsPsych trial object
  */
-export function buildCameraEndTrial() {
-  const recordingEndMarkup = h1(LANGUAGE.trials.camera.end);
-
-  return {
-    type: htmlKeyboardResponse,
-    stimulus: div(recordingEndMarkup),
-    trial_duration: 5000,
-    on_start: function () {
-      // Complete the camera recording
+const recordingEndMarkup = h1(LANGUAGE.trials.camera.end);
+export const buildCameraEndTrial = {
```

Comment:
The docstring above doesn't seem to match what it's supposed to be for. The docstring is just above `recordingEndMarkup` rather than `buildCameraEndTrial`